ray-operator/controllers/ray/common/pod.go

```diff
@@ -375,10 +356,8 @@ func BuildPod(podTemplateSpec v1.PodTemplateSpec, rayNodeType rayv1alpha1.RayNod
 	if podTemplateSpec.Annotations != nil {
 		if enabledString, ok := podTemplateSpec.Annotations[RayFTEnabledAnnotationKey]; ok {
 			if strings.ToLower(enabledString) == "true" {
-				// Ray FT is enabled and we need to add health checks
+				// If users do not specify probes, we will set the default probes.
 				if pod.Spec.Containers[rayContainerIndex].ReadinessProbe == nil {
-					// it is possible that some user have the probe parameters to override the default,
-					// in this case, this if condition is skipped
```

Comment:
Why remove this comment, is it inaccurate now?

Reply:
I didn't remove it. I simply moved it above the if statement that reads, `// If users do not specify probes, we will set the default probes.`.